test/cli/template-delete/complex/expected-stdout

```diff
@@ -0,0 +1,4 @@
+Plan for "delete-template" operation:
```

Comment:
Mozno `Plan for delete template instance operation:`? Neviem, co je viac zrozumitelne.